firefox_puppeteer/api/places.py

```diff
@@ -0,0 +1,156 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette.errors import TimeoutException
+
+from ..base import BaseLib
+
+
+class Places(BaseLib):
+    """Low-level access to several bookmark and history related actions."""
+
+    bookmark_folders = {'placesRoot': 1,
+                        'bookmarksMenuFolder': 2,
+                        'toolbarFolder': 3,
+                        'tagsFolder': 4,
+                        'unfiledBookmarksFolder': 5,
+                        }
```

Comment:
Sorry, but I still don't see what what you want here. Can you please give me an example? Not sure if its me or you who is confused, but what we need is that when you access e.g `bookmark_folder.places_root` it should be equivalent to `1`, because that is the internal id of this folder in Places.